javascript-timers/interval.js

```diff
@@ -0,0 +1,21 @@
+var $h1 = document.querySelector('h1');
+
+function countDown() {
+  if (timerID === 3) {
+    $h1.textContent = 3;
+  }
+  if (timerID === 2) {
+    $h1.textContent = 2;
+  }
+  if (timerID === 1) {
+    $h1.textContent = 1;
+  }
+  if (timerID === 0) {
+    $h1.textContent = '~Earth Beeeelooowww Us~';
+    clearInterval(intId);
+  }
+  timerID--;
```

Comment:
This is very manual. What if you want to make the countdown longer? Add an additional conditional for each number? You may want to use the counter variable as the text content, and if it is zero, you should set the text content to "~Earth Beeeelooowww Us~!"